server/models/db/workerTimeslot.js

```diff
@@ -21,7 +21,14 @@ async function updateWorkerAvailability(workerId, specificDate, newStatus) {
     return db.any(`update ${Tables.workerTimeslot} set status='${newStatus}' where date='${specificDate}' and worker_id='${workerId}';`);
 }
 
+//This method updates the worker avaiability table 
+async function updatingTheWorkerAvailability(status){
+    const queryCondition = `where w1.date='${specificDate}' and w1.worker_id='${workerId}'`;
+    return db.any(`update ${Tables.workerTimeslot} set status='${newStatus.unavailable}' where ;`)
+}
```

Comment:
Hi Simran,

This query statement updates the `status` to unavailable regardless of the initial value of the `status` parameter. I would recommend only changing the `status` to 'unavailable' if the original `status` was 'available'. If the status value is originally 'unavailable', there is no need to update the status parameter.

Reply:
Hi Nancy! 

Thanks for catching this! Peter and I have implemented changes in the latest PR to ensure that your comment can be resolved. Thank you very much Nancy! 

Simran

Reply:
Awesome! I'm glad you were able to address this!